SAPHana/SAPHanaSR-ScaleOut.spec

```diff
@@ -91,6 +91,8 @@ install -m 0444 man/*.8.gz %{buildroot}/usr/share/man/man8
 install -m 0555 bin/* %{buildroot}/usr/sbin
 install -m 0555 test/SAPHanaSR-replay-archive %{buildroot}/usr/sbin
 install -m 0555 test/SAPHanaSR-filter %{buildroot}/usr/sbin
+install -m 0555 test/ SAPHanaSR-SAPHanaSR-show-hadr-runtimes %{buildroot}/usr/sbin
```

Comment:
The blank between 'test/' and 'SAPHanaSR-SAPHanaSR-show-hadr-runtimes' is a problem and needs to be removed as it will prevent the package build.

Reply:
Will do that ASAP.